docs/book/src/reference/api/metadata-propagation.md

```diff
@@ -63,7 +63,10 @@ Top-level labels that meet a specific cretria are propagated to the Node labels
 - `.labels.[label-meets-criteria]` => `Node.labels`
 - `.annotations` => Not propagated.
 
-Label should meet one of the following criteria to propagate to Node: 
-- Has `node-role.kubernetes.io` as prefix.
-- Belongs to `node-restriction.kubernetes.io` domain.
-- Belongs to `node.cluster.x-k8s.io` domain.  
+Labels on a Machine that match the list of target domains provided by the combined list of the default sync machine label domains and `--additional-sync-machine-labels` on the manager are synced down to the node.
```

Comment:
If I'm not wrong it is not the combination of the list + the flag, but just the flag value.
If correct, please fix the comment

Reply:
It is the combined list. Ref for details: https://github.com/kubernetes-sigs/cluster-api/pull/11650#discussion_r1915518213